scripts/match.gd

```diff
@@ -178,5 +200,36 @@ func set_server_to(server):
 		serve_marker[0].hide()
 		serve_marker[1].show()
 
+## Play a sound by key, first checking if there is an existing wav to use,
+## and if not then attempting to use TTS as a fallback.
+func play_sound(key:String) -> void:
+	if sounds_dict.has(key) and sounds_dict[key] is AudioStreamWAV:
+		$AudioStreamPlayer.stream = sounds_dict[key]
+		$AudioStreamPlayer.play()
+	else:
+		Global.speak_tts(key)
+
+## Queue the playing of a sound which is specified by sound key
+func queue_sound(key:String) -> void:
```

Comment:
just added this sound queue system, so just use `queue_sound` instead of `play_sound`